public/editor/scripts/main.js

```diff
@@ -89,6 +89,7 @@ require(["jquery", "bowser", "analytics"], function($, bowser, analytics) {
     var appUrl = thimbleScript.getAttribute("data-app-url");
     var projectDetails = thimbleScript.getAttribute("data-project-details");
     var editorUrl = thimbleScript.getAttribute("data-editor-url");
+    clearTimeout(slowLoadingTimeout);
```

Comment:
I wonder if this is the right place to do this, since this doesn't represent a point where the editor is loaded or usable.  Some other options might include:

* Once Bramble fires the `ready` event here https://github.com/mozilla/thimble.mozilla.org/blob/master/public/editor/scripts/editor/js/bramble-editor.js#L34
* Once the editor is usable, and we're about to remove the spinner/loader here https://github.com/mozilla/thimble.mozilla.org/blob/master/public/editor/scripts/editor/js/fc/bramble-ui-bridge.js#L446

Doing that would require us to move this `timeout` to a separate module, so you can `require` it in a few places and start/stop it.